lte/gateway/deploy/roles/magma/tasks/main.yml

```diff
@@ -67,9 +67,18 @@
   copy: src=magma-preferences dest=/etc/apt/preferences.d/magma-preferences
   when: full_provision
 
-- name: Copy sysctl file for core dumps
-  copy: src=99-magma.conf dest=/etc/sysctl.d/99-magma.conf
-  when: full_provision
+- name: Set core pattern
+  sysctl:
+    name: kernel.core_pattern
+    value: '|/usr/local/bin/coredump core-%t-%e-%p'
+    state: present
```

Comment:
this already set in sysctl conf file, lets use it rather using sysclt cmd.

`sudo sysctl --system`
or
`sudo sysctl -p /etc/sysctl.d/99-magma.conf`

Reply:
ok let me try that